js/index.js

```diff
@@ -1,7 +1,78 @@
 // Iteration 1: Names and Input
 
+let hacker1 = "Nora";
+//console.log(`The driver's name is ${hacker1}`);
+let hacker2 = "Jason";
+//console.log(`The navigators's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `The driver has the longest name, it has ${hacker1.length} characters`
+  );
+} else if (hacker1.length < hacker2.length) {
+  console.log(
+    `It seems that the navigator has the longest name, it has ${hacker2.length} characters`
+  );
+} else {
+  console.log(
+    `Wow, you both have equally long names, ${hacker2.length} characters!`
+  );
+}
 
-// Iteration 3: Loops
\ No newline at end of file
+let upperCaseName = "";
+for (let i = 0; i < hacker1.length; i++) {
+  upperCaseName += hacker1[i].toUpperCase() + " ";
+}
+console.log(upperCaseName);
+
+console.log(
+  hacker2
+    .split("")
+    .reverse()
+    .join("")
+);
+
+let reverseName = "";
+for (let i = hacker2.length - 1; i >= 0; i--) {
+  reverseName += hacker2[i];
+}
+console.log(reverseName);
+
+// Iteration 3: Loops
+
+if (hacker1 < hacker2) {
```

Comment:
I would be careful to do this when you want to sort by lexicographical order because it does work for every case. Best is to use localeCompare()